viite-UI/src/view/TileMapCollection.js

```diff
@@ -80,11 +80,11 @@
       propertyBorder: propertyBorderLayer
     };
 
-    if (OAGGreyConfig) {
-      var parser = new ol.format.WMTSCapabilities();
-      var result = parser.read(OAGGreyConfig);
-      var config = {layer: 'liikennevirasto:PTP_Taustakartta_Harmaa', matrixSet: 'EPSG:3067_PTP_JHS180'};
-      var options = ol.source.WMTS.optionsFromCapabilities(result, config);
+    var parser = new ol.format.WMTSCapabilities();
+    var result = parser.read(OAGGreyConfig);
+    var config = {layer: 'liikennevirasto:PTP_Taustakartta_Harmaa', matrixSet: 'EPSG:3067_PTP_JHS180'};
+    var options = ol.source.WMTS.optionsFromCapabilities(result, config);
+    if (options) {
```

Comment:
Mihin näitä greylayereita ja greyconfigia käytetään?

Reply:
Ei niitä oikeastaan enää käytetä, voidaan myös poistaa